staging/src/k8s.io/client-go/examples/third-party-resources/main.go

```diff
@@ -20,17 +20,26 @@ package main
 import (
 	"flag"
 	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/golang/glog"
```

Comment:
Thanks for the clue.  I am not familiar with bazel and that script; where would I go to learn what they do?

I tried that command, and it failed as follows.  What does it mean and what can I do about it?
```
mjs10:kubernetes mspreitz$ ./hack/update-bazel.sh 
F0503 13:04:03.982558    3351 gazel.go:60] err walking generated: openapi-gen path outside of kubernetes: github.com/MikeSpreitzer/kubernetes/cmd/libs/go2idl/client-gen/test_apis/testgroup/v1
```